File: eftl-dotnet-sdk/TIBCO.EFTL.WINRT/websocket/WebSocketException.cs
/*
 * Copyright (c) 2013-$Date: 2016-01-25 19:33:46 -0600 (Mon, 25 Jan 2016) $ TIBCO Software Inc.
 * Licensed under a BSD-style license. Refer to [LICENSE]
 * For more information, please contact:
 * TIBCO Software Inc., Palo Alto, California, USA
 *
 * $Id: WebSocketException.cs 83756 2016-01-26 01:33:46Z bmahurka $
 *
 */
using System;

namespace TIBCO.EFTL
{
    public class WebSocketException: Exception 
    {
        public WebSocketException(String message): base(message) {}
    }
}

File: eftl/sdk/java/src/com/tibco/eftl/websocket/WebSocketException.java
/*
 * Copyright (c) 2013-$Date: 2013-12-04 09:51:34 -0800 (Wed, 04 Dec 2013) $ TIBCO Software Inc.
 * Licensed under a BSD-style license. Refer to [LICENSE]
 * For more information, please contact:
 * TIBCO Software Inc., Palo Alto, California, USA
 *
 * $Id: WebSocketException.java 70851 2013-12-04 17:51:34Z $
 *
 */
package com.tibco.eftl.websocket;

@SuppressWarnings("serial")
public class WebSocketException extends Exception {

    public WebSocketException(String message) {
        super(message);
    }
}
